rllib/agents/pg/torch_pg_policy.py

```diff
@@ -3,38 +3,37 @@
 from __future__ import print_function
 
 import ray
-from ray.rllib.evaluation.postprocessing import compute_advantages, \
-    Postprocessing
+from ray.rllib.agents.pg.tf_pg_policy import post_process_advantages
```

Comment:
Hm should we be importing tf policy from torch files? I guess it's ok as long as it doesn't fail if tensorflow isn't installed.

Reply:
It won't fail b/c we are using try_import_tf(). And the function itself that we are importing has no tf-dependency in it.
But it's a valid question: Should this function be in the tf-specific file then?